DQMOffline/Trigger/python/SiPixel_OfflineMonitoring_cff.py

```diff
@@ -2,8 +2,15 @@
 
 from DQMOffline.Trigger.SiPixel_OfflineMonitoring_Cluster_cff import *
 from DQMOffline.Trigger.SiPixel_OfflineMonitoring_TrackCluster_cff import *
+from RecoPixelVertexing.PixelLowPtUtilities.siPixelClusterShapeCache_cfi import *
+from DQM.SiPixelMonitorTrack.RefitterForPixelDQM import *
 
+hltSiPixelClusterShapeCache = siPixelClusterShapeCache.clone(src = 'hltSiPixelClusters')
+hltrefittedForPixelDQM = refittedForPixelDQM.clone(src ='hltMergedTracks',
+                                                   TTRHBuilder = cms.string('WithTrackAngle')) # no templates at HLT
```

Comment:
```suggestion
                                                   TTRHBuilder = 'WithTrackAngle') # no templates at HLT
```

Reply:
to speed up things here's the fix in master: https://github.com/cms-sw/cmssw/pull/40891.
Hopefully @arossi83 can do the same in https://github.com/cms-sw/cmssw/pull/40875.